### PR TITLE
Use workflow helpers for trigger finished webhook

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

## This introduces a breaking change

- [x] Yes
- [ ] No

## This Pull Request implements

Update trigger finished wehook removing Python script and using workflow-helpers image

### CODE REVIEW

File: adviser/base/argo-workflows/finished-webhook.yaml [hunk 1]
```diff
@@ -11,84 +11,16 @@ spec:
...
-              WEBHOOK_CALLBACK_URL = os.environ["WEBHOOK_CALLBACK_URL"]
-
-              response = requests.post(WEBHOOK_CALLBACK_URL, data=json.dumps(data), headers=headers)
-              response.raise_for_status()
-            else:
-              print("There are missing values for GitHub App", metadata)
-          else:
-              print("There are missing keys for GitHub App", metadata)
-              print("Thamos has run through CLI")
+        image: workflow-helpers
```
Comment (goern) [inline adviser/base/argo-workflows/finished-webhook.yaml:16]: we need to have an imagestreamtag object for this

Comment (pacospace) [reply adviser/base/argo-workflows/finished-webhook.yaml:16]: Updated in 8a5ea93420c073f3c0eb65272375d5a2e156880c

**Pull request merged**